client/webserver/site/src/js/orders.js

```diff
@@ -179,6 +183,40 @@ export default class OrdersPage extends BasePage {
     return res.orders
   }
 
+  /* exportOrders open a new page to export orders based on the current filter */
+  exportOrders () {
+    const page = this.page
+    this.offset = ''
+    const filterState = this.filterState
+    filterState.hosts = parseSubFilter(page.hostFilter)
+    filterState.assets = parseSubFilter(page.assetFilter)
+    filterState.statuses = parseSubFilter(page.statusFilter)
+
+    const url = new URL(window.location)
+    const search = new URLSearchParams(url.search)
+    search.delete('offset')
```

Comment:
Is the idea because the _export_ http handler only recognizes `'hosts'`, `'assets'`, and `'statuses'`?  If so, that means it doesn't use `'offset'` **or** `'n'`.  It does seem to be used in `exportOrders` via `window.open(url.toString())`.  Anyway, that's not the case in `submitFilter`, where it's doing a POST with `currentFilter` to get the request body... @buck54321 maybe you had originally written `submitFilter` using url query parameters instead of a POST?